python/TestHarness/TestHarness.py

```diff
@@ -1144,7 +1144,7 @@ def parseCLArgs(self, argv):
         # Options that pass straight through to the executable
         parser.add_argument('--parallel-mesh', action='store_true', dest='parallel_mesh', help='Deprecated, use --distributed-mesh instead')
         parser.add_argument('--distributed-mesh', action='store_true', dest='distributed_mesh', help='Pass "--distributed-mesh" to executable')
-        parser.add_argument('--device', action='store', dest='device', type=str, choices=TestHarness.validDevices(), default='cpu', help='Run libtorch or MFEM tests with this device; device availability depends on library support and compilation settings')
+        parser.add_argument('--compute-device', action='store', dest='device', type=str, choices=['cpu', 'cuda', 'hip', 'mps'], default='cpu', help='Run libtorch or MFEM tests with this device; device availability depends on library support and compilation settings')
```

Comment:
Don't overwrite my `TestHarness.validDevices()` change plz. (You could rename the method to `validComputeDevices()` if you wish). This makes it slightly easier because then there's only one place in our python code a developer has to add a new device. 

If you want to switch to `--compute-device` for the test harness and delete `--device`, we'll need to change the civet recipe. To not break everyone's PRs it might be best to leave `--device` for now. And then we can wait at least until this PR makes it to devel and then we can change the civet recipe. Finally we can come back and remove `--device` from the test harness

Reply:
Yeah, already done, just not pushed